src/browser.js

```diff
@@ -18,7 +18,7 @@ const validator = new LaravelValidator;
  * @param {String} result.suggestion - A suggestion for text to replace form input with.
  */
 function showValidationMessage($field, result) {
-  console.log($field, result);
+  // console.log($field, result);
```

Comment:
Yeah, wasn't sure. Was getting a bunch of output and didn't know if intended for the time being or not @_@